doc/vaulted-edit.1.md

```diff
@@ -27,8 +27,10 @@ AWS KEY
    Manages MFA (multi-factor authentication) ARN or serial number. If enabled, user will be prompted
    to enter MFA code when accessing vault. MFA will remain active for the
    vault duration.
+   If you are using STS credentials (see the 't' option below), you must have MFA enabled to invoke any IAM calls.
 * r - Role  
    ARN of role to be assumed when accessing vault.
+   When assuming a role, the maximum duration allowed is 1 hour. If your duration is greater than 1 hour when setting a role, the duration will adjusted to this point.
```

Comment:
The "to this point." at the end was slightly confusing to my brain on first read. Perhaps we can word this differently?